core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -375,6 +375,14 @@
           + "first delayed, then throttled. Default is 25.";
   private static final int DOS_FILTER_MAX_REQUESTS_PER_SEC_DEFAULT = 25;
 
+  private static final String DOS_FILTER_MAX_REQUESTS_GLOBAL_PER_SEC_CONFIG =
+      "dos.filter.max.requests.global.per.sec";
+  private static final String DOS_FILTER_MAX_REQUESTS_GLOBAL_PER_SEC_DOC =
+      "Maximum number of requests per second for the rest instance. Requests in excess of this are "
+          + "first delayed, then throttled. If set to the default value of -1, then the value of "
+          + "dos.filter.max.requests.per.sec is used.";
+  private static final int DOS_FILTER_MAX_REQUESTS_GLOBAL_PER_SEC_DEFAULT = -1;
```

Comment:
I think we should:

 * Remove `DOS_FILTER_REMOTE_PORT_CONFIG`
 * Remove `DOS_FILTER_TRACK_GLOBAL_CONFIG`.
 * Add `DOS_FILTER_PER_CONNECTION_ENABLED_CONFIG`
 * Add `DOS_FILTER_PER_CONNECTION_MAX_REQUESTS_PER_SECOND_CONFIG`
 * Profit